ote/src/cljs/ote/views/parking.cljs

```diff
@@ -86,7 +86,7 @@
 
 (defn service-hours-group [e!]
   (let [tr* (tr-key [:field-labels :service-exception])
-        write (fn [key]
+        write-time (fn [key]
                 (fn [{all-day? ::t-service/all-day :as data} time]
```

Comment:
all-day? is also used in rental form.

Reply:
Yes. Rental uses the same common component that passenger-transportation uses